### DISCUSSION TITLE
The IoTy Project

### DISCUSSION THREAD

**Discussion by QuirkyCort** [BIPES/BIPES#203] [General]
Hi. I'm the developer for the IoTy project (https://github.com/QuirkyCort/IoTy). I started it about half a year ago to facilitate teaching electronics and Internet-of-Things development to kids and teens. Didn't know about the BIPES project until now, or I may not have bothered starting on mine.

Anyway, I've just dropped by to say hi, and hope that we'll be able to learn from each other in the future.

**Comment by rafaelaroca** [comment]
Dear @QuirkyCort, thank you for letting us know about IoTy, and congratulations.

THe UI/UX is so much clean and beautiful!! Very nice!

As you said, I also hope that we could lear from each other and both projects grow! 

best regards!
Rafael
